composer.json

```diff
@@ -3,8 +3,8 @@
     "description": "Nextcloud coding standards for the php cs fixer",
     "type": "library",
     "require": {
-        "php": "^7.3|^8.0",
-        "friendsofphp/php-cs-fixer": "^3.2"
+        "php": "^7.4|^8.0",
```

Comment:
We need 7.3 so we can still use it in branches that support 21/22?

Reply:
I think that the best way on branches <= 22 is to don't bump the next version of this package.

Maybe will be good put on README of this repository a compatibility table with two columns, one for server version and other for this package version. Example:
| Nextcloud Server | coding-standard |
| ---------------- | --------------- |
| <= stable22         | 1.0             |
| >= stable23         | 1.1             |